selfdrive/car/subaru/carcontroller.py

```diff
@@ -32,13 +37,25 @@ def update(self, CC, CS, now_nanos):
       new_steer = int(round(apply_steer))
       apply_steer = apply_driver_steer_torque_limits(new_steer, self.apply_steer_last, CS.out.steeringTorque, self.p)
 
+      apply_steer_req = 1
+
       if not CC.latActive:
         apply_steer = 0
+        apply_steer_req = 0
+
+      if self.CP.carFingerprint in STEER_LIMITED_2020:
+        # Steering rate fault prevention
+        self.steer_rate_counter, apply_steer_req = \
+          common_fault_avoidance(CS.out.steeringRateDeg, MAX_STEER_RATE, apply_steer_req,
+                                 self.steer_rate_counter, MAX_STEER_RATE_FRAMES)
+        
+        # >90 degree steering fault prevention
+        _, apply_steer_req = common_fault_avoidance(CS.out.steeringAngleDeg, MAX_STEER_ANGLE, apply_steer_req, 0, 0, 0)
```

Comment:
Let's only handle one fault at a time

Reply:
removed 90 deg prevention, we can investigate that later when it becomes more relevant when we get subaru long control